app/models/log.rb

```diff
@@ -40,7 +40,11 @@ def collection_start_year
   end
 
   def lettings?
-    false
+    is_a? LettingsLog
```

Comment:
What's the motivation for this change? I somewhat prefer the inheritance here vs type-checking, otherwise `Log` needs to have knowledge of its children.